apps/system/js/nfc_handover_manager.js

```diff
@@ -250,8 +186,7 @@
           Service.query('BluetoothTransfer.isSendFileQueueEmpty')) {
         if (this.bluetoothAutoEnabled) {
           this._debug('Disabling Bluetooth');
-          // XXX: Let BluetoothCore do this for you
-          this.writeSetting({'bluetooth.enabled': false});
+          window.dispatchEvent(new CustomEvent('request-disable-bluetooth'));
```

Comment:
above Service.query('BluetoothTransfer.isSendFileQueueEmpty') denotes Bluetooth module is ready, or the code wont flow into here